webhook_features/README.md

```diff
@@ -0,0 +1,8 @@
+# Drahtbot
+
+## Run
+
+```bash
+$ cargo build --release
+$ ./target/release/drahtbot --repo bitcoin/bitcoin --repo bitcoin-core/gui --token $GITHUB_TOKEN --bot-username DrahtBot
```

Comment:
```suggestion
$ ./target/release/drahtbot --repo bitcoin/bitcoin --repo bitcoin-core/gui --token $GITHUB_TOKEN
```

can be removed now?

Reply:
I just fixed a bug where the bot would run twice because it would receive a webhook event of its own comment edition after someone comments an ACK, so it would re-run the parsing etc.
https://github.com/MarcoFalke/DrahtBot/compare/e3ba4dac07cfe6b9f65f5d645c7b13a87a9ea597..73c373b81f7d69d6982e49f60a17007b4c1e5905

So I think the bot username is needed, but can be deduced at runtime with an API req if you prefer.

Reply:
Removed `--repo` and now deduce username at runtime.